tsconfig.json

```diff
@@ -6,7 +6,9 @@
     "rootDir": "./src",
     "strict": true,
     "noImplicitAny": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "sourceMap": true,
```

Comment:
Consider setting `sourceMap` to `false` if `inlineSourceMap` is intended to be used. Having both `sourceMap` and `inlineSourceMap` enabled can lead to confusion and potentially larger output files.